CoffeeShop/src/main/java/item/ItemList.java

```diff
@@ -0,0 +1,113 @@
+package item;
+
+import interfaces.EntityList;
+import java.util.HashMap;
+import utils.ItemCategory;
+import exceptions.DuplicateItemIDException;
+
+/**
+ * @author Fraser Holman
+ *
+ * Class represents a list of all avaible items that can be ordered
+ *
+ * Contains a hashmap of all the items
+ */
+
+public class ItemList implements EntityList<Item, String> {
+    private HashMap<String, Item> items;
+
+    /**
+     * Initialises the hashmap to contain the items
+     */
+    public ItemList() {
+        items = new HashMap<String, Item>();
+    };
+
+    /**
+     * Adds an item to the hashmap
+     *
+     * @param item The item to be added to the hashmap
+     */
+    @Override
+    public void add(Item item) throws DuplicateItemIDException {
+        if (items.containsKey(item.getItemID())) {
+            throw new DuplicateItemIDException("Item ID is not unique");
+        }
+        else items.put(item.getItemID(), item);
+    }
```

Comment:
```suggestion
    @Override
    public void add(Item item) throws DuplicateItemIDException {
        if (items.containsKey(item.getItemID())) {
            throw new DuplicateItemIDException("Item ID is not unique");
        }
        
        items.put(item.getItemID(), item);
    }
```

Just a lil suggestion, what you have with the if statement is called a Guard Clause so we don't need to explicitly state the else here just to make the code cleaner